sim/rogue/talents.go

```diff
@@ -96,8 +96,8 @@ func (rogue *Rogue) registerColdBloodCD() {
 			}
 		},
 		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
-			// for Fan of Knives and Mutilate, the offhand hit comes first and is ignored, so the aura doesn't fade too early
-			if spell.Flags.Matches(SpellFlagColdBlooded) && spell.ProcMask.Matches(core.ProcMaskMeleeMH) {
+			// for Mutilate, the mainhand hit comes first and is ignored, so the aura doesn't fade too early
+			if spell.Flags.Matches(SpellFlagColdBlooded) && spell.ProcMask.Matches(core.ProcMaskMeleeOH) {
```

Comment:
Conditional should be adjusted to include SpellID of Mutilate OH since there are now several offhand abilities that should be affected by Cold Blood. These include Shiv, Poisoned Knife, and Main Gauche.

Reply:
This is for disabling the aura after a "cold blooded" ability used it, so Shiv & friends would all still profit. But it's still wrong, since those spells now profit, but don't disable Cold Blood anymore.

Reply:
Mutilate Hit order could be reversed again if that would fix this issue.